devices/immax.js

```diff
@@ -148,10 +148,14 @@ module.exports = [
         fromZigbee: [fz.battery, fz.ignore_basic_report, fz.illuminance, fz.ZB003X, fz.ZB003X_attr, fz.ZB003X_occupancy],
         toZigbee: [tz.ZB003X],
         exposes: [e.occupancy(), e.tamper(), e.battery(), e.illuminance(), e.illuminance_lux().withUnit('lx'), e.temperature(),
-            e.humidity(), exposes.numeric('reporting_time', ea.STATE_SET).withDescription('Reporting interval in minutes'),
-            exposes.numeric('temperature_calibration', ea.STATE_SET).withDescription('Temperature calibration'),
-            exposes.numeric('humidity_calibration', ea.STATE_SET).withDescription('Humidity calibration'),
-            exposes.numeric('illuminance_calibration', ea.STATE_SET).withDescription('Illuminance calibration'),
+            e.humidity(), exposes.numeric('reporting_time', ea.STATE_SET).withDescription('Reporting interval in minutes')
+                .withValueMin(0).withValueMax(1440),
+            exposes.numeric('temperature_calibration', ea.STATE_SET).withDescription('Temperature calibration')
```

Comment:
I'm confused, is this a internal device thing? Or should these really not be expose data but options ?

Reply:
this device support calibration on the device side (instead of on the z2m side as usual)